grafana/k6-docs
# Feat/collapsible sections

This PR adds `Collapsible` component and documentation for it.

Steps to test:
1. Run the project locally
2. Add `Collapsible` to one of `md` files. Use the following syntax:

```md


<Collapsible title="Collapsible title">

</Collapsible>


```
3. Open the corresponding page in the browser and confirm that everything looks and works as expected.
